Src/CSharpier.Tests/TestFiles/IfStatement/IfStatements.cst

```diff
@@ -20,13 +20,26 @@ public class ClassName
             return;
         }
 
-        if (true)
-            DoSoemthing();
-        else if (false)
-            DoSomething();
+        if (true) DoSomething();
+        else if (false) DoSomething();
         else
             DoSomething;
 
+        if (someLongCondition != someOtherCondition)
+            DoSomeLongMethodCall111111111111();
+        else if (someLongCondition == someOtherCondition)
+            DoSomeLongMethodCall111111111111();
+        else
+            DoSomeLongMethodCall1111111111111111111111111111111111111111();
```

Comment:
Not really a point, but I really like padding length by adding more `o`s to `Long` than adding `1`s to the end.
That way, when I pronounce it in my head as a really loooooooong method, I _feel_ the weight of the function call.

Reply:
I was on board with it..... except rider complains about LooooooooooongMethod() being misspelled and seeing things underlined with red triggers my OCD.
How about ending with ________________ like this? It doesn't have the weight, but I think it makes it easier to read without getting distracted by all the extra 11111111111111111111s or the red squiggles